### PR TITLE
Blog: Marketing Tips an Open Source Project: Tips from a First-Timer

### PR DESCRIPTION
**Pull request by dannac2**
# Docs updates should not be submitted in this repository

Our doc site content is being automatically updated from [Wing](https://github.com/winglang/wing) repository docs folder.

Please submit any changes to the docs as a PR [here](https://github.com/winglang/wing/pulls)

### CODE REVIEW

File: blog/2023-09-19-marketing-tips-for-open-source.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+tags: [cloud-oriented programming, winglang, marketing]
+hide_table_of_contents: true
+---\
+
+## Marketing Tips an Open Source Project: Tips from a First-Timer
+
+Hey there! As a marketer with nearly a decade of experience in the fields of FinTech, MedTech, AI, and Cybersecurity, this was my first time working on an open source project. Turns out, gaining devs' trust is like playing a whole new game. So, I decided to share what I have learned so far in the world of marketing an open source project. 
+
+### Elevate Your Documentation: 
+Okay, I can't stress this enough. Open source thrives on community contributions. If they can't understand your project, they can't contribute. This is probably one of the biggest lessons I have learned so far. And the best method to succeed? Ensure your documentation is clear, engaging, and transparent. Think of it as a user manual that constantly needs to be maintained.
```
Comment (revitalbarletz) [inline blog/2023-09-19-marketing-tips-for-open-source.md:15]: Why question mark after the succeed?

File: blog/2023-09-19-marketing-tips-for-open-source.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - dannac2
+tags: [cloud-oriented programming, winglang, marketing]
+hide_table_of_contents: true
+---\
+
+## Marketing Tips an Open Source Project: Tips from a First-Timer
+
+Hey there! As a marketer with nearly a decade of experience in the fields of FinTech, MedTech, AI, and Cybersecurity, this was my first time working on an open source project. Turns out, gaining devs' trust is like playing a whole new game. So, I decided to share what I have learned so far in the world of marketing an open source project. 
+
+### Elevate Your Documentation:
```
Comment (eladb) [inline blog/2023-09-19-marketing-tips-for-open-source.md:14]: Nit: Remove `:` from headings.

```suggestion
### Elevate Your Documentation
```

**Review by eladb — COMMENTED** [review]
I like this!

Any chance you can add a small example from our project in each one of these tips?

File: blog/2023-09-19-marketing-tips-for-open-source.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Marketing Tips an Open Source Project: Tips from a First-Timer
```
Comment (ShaiBer) [inline blog/2023-09-19-marketing-tips-for-open-source.md:2]: The phrasing of the title feels a bit off to me

**Comment by dannac2** [comment]
@ShaiBer @eladb so...I didn't really like my original version lol, and I decided to re-write the last few tips. I want to add examples as images, but i seem to have trouble with the code. What should I use to insert an image in the body after I upload it to the assets?

**Pull request merged**